include/ocpp/v16/charge_point.hpp

```diff
@@ -436,6 +436,12 @@ class ChargePoint {
     void register_configuration_key_changed_callback(const CiString<50>& key,
                                                      const std::function<void(const KeyValue& key_value)>& callback);
 
+    /// \brief registers a \p callback function that can be used to react on changed configuration key. This
```

Comment:
Please adjust the method docs so to point out that this is a generic callback and that it will only be called in case no specific callback for the key is registered.